### ISSUE TITLE
InexactError on Julia 0.6 32-bit Windows

### ISSUE THREAD

**Issue by ararslan** [JuliaStats/StatsBase.jl#288]
From the AppVeyor log:
```
histrange: Error During Test
  Got an exception of type InexactError outside of a @test
  InexactError()
  Stacktrace:
   [1] trunc at .\float.jl:651
   [2] floatrange(::Float64, ::Float64, ::Float64, ::Float64) at .\twiceprecision.jl:111
   [3] histrange(::Float64, ::Float64, ::Int32, ::Symbol) at C:\Users\appveyor\.julia\v0.6\StatsBase\src\hist.jl:118
   [4] histrange(::Array{Int32,1}, ::Int32, ::Symbol) at C:\Users\appveyor\.julia\v0.6\StatsBase\src\hist.jl:54
   [5] macro expansion at C:\Users\appveyor\.julia\v0.6\StatsBase\test\hist.jl:107 [inlined]
   [6] macro expansion at .\test.jl:860 [inlined]
   [7] macro expansion at C:\Users\appveyor\.julia\v0.6\StatsBase\test\hist.jl:76 [inlined]
   [8] macro expansion at .\test.jl:860 [inlined]
   [9] anonymous at .\<missing>:?
   [10] include_from_node1 at .\loading.jl:569
   [11] include at .\sysimg.jl:14
   [12] macro expansion at C:\Users\appveyor\.julia\v0.6\StatsBase\test\runtests.jl:27 [inlined]
   [13] anonymous at .\<missing>:?
   [14] include_from_node1 at .\loading.jl:569
   [15] include at .\sysimg.jl:14
   [16] process_options at .\client.jl:305
   [17] _start at .\client.jl:371
Test Summary:                      | Pass  Error  Total
StatsBase.Histogram                |   87      1     88
  Histogram binindex and binvolume |    7             7
  Histogram append                 |    5             5
  Histogram fit                    |   19            19
  Histogram element type           |    4             4
  histrange                        |   22      1     23
  Histogram show                   |    4             4
  Histogram norm and normalize     |   19            19
  Histogram zero                   |    5             5
  Histogram merge                  |    2             2
ERROR: LoadError: LoadError: Some tests did not pass: 87 passed, 0 failed, 1 errored, 0 broken.
while loading C:\Users\appveyor\.julia\v0.6\StatsBase\test\hist.jl, in expression starting on line 6
while loading C:\Users\appveyor\.julia\v0.6\StatsBase\test\runtests.jl, in expression starting on line 24
```
Not sure whether it's our problem or a problem in Base, haven't looked into it.

**Comment by nalimilan** [comment]
I've mentioned this elsewhere and filed an issue in Base about this a few weeks ago.

**Comment by andreasnoack** [comment]
Ref: https://github.com/JuliaLang/julia/pull/22644

**Comment by ararslan** [comment]
Sorry, I didn't realize this had been reported in Base. I guess this should be fixed in 0.6.1 once the linked PR is backported.
